app/controllers/tokens_controller.rb

```diff
@@ -28,4 +29,8 @@ def set_token
   def set_team_email
     ENV['TEAM_EMAIL'] || 'nomisapi@digital.justice.gov.uk'
   end
+
+  def set_parsed_permissions
+    @token.permissions.split.map { |p| Regexp.new(p).inspect }
```

Comment:
This causes an error. `eval(@token.permissions)` instead works fine.

<img width="1050" alt="screen shot 2017-10-18 at 15 27 35" src="https://user-images.githubusercontent.com/4628936/31724172-ef876d4c-b418-11e7-8691-ca7e1da6feee.png">

Reply:
is it possible the permissions are malformed there?

Reply:
i.e. is the square bracket in ...`char-class: /["`... supposed to be there?

Reply:
you might need to reset your DB

Reply:
This was a new Token record I created to test with.